src/EnergyPlus/MixedAir.cc

```diff
@@ -142,15 +142,15 @@ using namespace FaultsManager;
 
 constexpr std::array<std::string_view, static_cast<int>(ControllerKind::Num)> ControllerKindNamesUC{"CONTROLLER:WATERCOIL", "CONTROLLER:OUTDOORAIR"};
 
-Array1D_string const CurrentModuleObjects(8,
-                                          {"AirLoopHVAC:OutdoorAirSystem",
-                                           "AirLoopHVAC:OutdoorAirSystem:EquipmentList",
-                                           "AirLoopHVAC:ControllerList",
-                                           "AvailabilityManagerAssignmentList",
-                                           "Controller:OutdoorAir",
-                                           "ZoneHVAC:EnergyRecoveryVentilator:Controller",
-                                           "Controller:MechanicalVentilation",
-                                           "OutdoorAir:Mixer"});
+const std::array<std::string_view, static_cast<int>(CMO::Num)> CurrentModuleObjects{"None",
+                                                                                    "AirLoopHVAC:OutdoorAirSystem",
+                                                                                    "AirLoopHVAC:OutdoorAirSystem:EquipmentList",
+                                                                                    "AirLoopHVAC:ControllerList",
+                                                                                    "AvailabilityManagerAssignmentList",
+                                                                                    "Controller:OutdoorAir",
+                                                                                    "ZoneHVAC:EnergyRecoveryVentilator:Controller",
+                                                                                    "Controller:MechanicalVentilation",
+                                                                                    "OutdoorAir:Mixer"};
```

Comment:
constexpr here and const in `.hh.? or constexpr both places?

Reply:
`const` in the `.hh`. In order for something to be `constexpr` it has to have the definition, i.e., the actual value.